submission/persistence/src/main/kotlin/ac/uk/ebi/biostd/persistence/mapping/extended/from/ToDbSection.kt

```diff
@@ -0,0 +1,58 @@
+package ac.uk.ebi.biostd.persistence.mapping.extended.from
+
+import ac.uk.ebi.biostd.persistence.common.NO_TABLE_INDEX
+import ac.uk.ebi.biostd.persistence.model.Section
+import ac.uk.ebi.biostd.persistence.model.SectionAttribute
+import ac.uk.ebi.biostd.persistence.model.Submission
+import arrow.core.Either
+import arrow.core.Either.Left
+import arrow.core.Either.Right
+import ebi.ac.uk.extended.model.ExtAttribute
+import ebi.ac.uk.extended.model.ExtSection
+import ebi.ac.uk.extended.model.ExtSectionTable
+import java.util.SortedSet
+
+internal fun ExtSection.toDbSection(submission: Submission, index: Int): Section {
```

Comment:
the problem is that this function is an extension so `this` is already `ExtSection` so if we add `apply` this will become `Section` and as name fields are the same i.e `sections` names with a crash (which we can solve with ` sections = this@toDbSection.sections.toDbSections(submission)`) but that is more verbose.